code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm

```diff
@@ -479,11 +479,6 @@
 		M.overeatduration += 20 //Eating more than 10u will make you fat, 10u will give 500 overeatduration
 		..()
 
-/datum/reagent/lube/reaction_turf(turf/simulated/T, reac_volume)
```

Comment:
I hadn't noticed before but this seems to be an error in copypasta. Fat, if not cfat, should have a makeslippery effect.

Can you just fix the typepath on this proc definition please?